run/convert.go

```diff
@@ -59,9 +59,12 @@ func makeData(log *log.Logger, info *database.Info, cfg *Config) (*data.DBData,
 		}
 		for _, t := range s.Tables {
 			table := &data.Table{
-				DBName:        t.Name,
-				Schema:        sch,
-				ColumnsByName: make(map[string]*data.Column, len(t.Columns)),
+				DBName:                             t.Name,
+				Schema:                             sch,
+				ColumnsByName:                      make(map[string]*data.Column, len(t.Columns)),
+				ForeignColumnsByForeignKey:         map[string][]*data.ForeignColumn{},
+				ForeignTablesByForeignKey:          map[string]*data.ForeignTable{},
+				ForeignTablesByForeignKeyReference: map[string]*data.ForeignTable{},
```

Comment:
Let's not make people spell foreign... how about this?

FKColsByFK
FKTablesByFK
FKTablesByFKRef

FK is a pretty bog standard abbreviation in the SQL world.

Reply:
but see below, not sure we need the tables.